mdata/store_cassandra.go

```diff
@@ -22,19 +23,21 @@ import (
 const Month_sec = 60 * 60 * 24 * 28
 
 const keyspace_schema = `CREATE KEYSPACE IF NOT EXISTS %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}  AND durable_writes = true`
-const table_schema = `CREATE TABLE IF NOT EXISTS %s.metric (
+const table_schema = `CREATE TABLE IF NOT EXISTS %s.%s (
     key ascii,
     ts int,
     data blob,
     PRIMARY KEY (key, ts)
 ) WITH CLUSTERING ORDER BY (ts DESC)
-    AND compaction = {'class': 'org.apache.cassandra.db.compaction.TimeWindowCompactionStrategy', 'compaction_window_unit': 'DAYS', 'compaction_window_size': '1' }
-    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}`
+    AND compaction = {'class': 'org.apache.cassandra.db.compaction.TimeWindowCompactionStrategy', 'compaction_window_unit': 'HOURS', 'compaction_window_size': '%d' }
```

Comment:
The full class path should only be used for custom classes.   The in-built ones should use the short name.  I don't really know why it is just how it is presented in the latest docs.

https://docs.datastax.com/en/cassandra/3.0/cassandra/operations/opsConfigCompress.html?hl=compression